integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -670,6 +670,12 @@ csi:
 
 hub:
   enabled: false
+  clusterId:
+  elasticsearch:
+    endpoint: "endpoint"
+    port: "port"
+    username: "username"
+    password: "password"
```

Comment:
If there's no good default value, for dummy values could you just leave it as `""` with a comment giving example configuration values? See some other sections in this Helm chart for an example.

Reply:
updated